### ISSUE TITLE
Update Friendly Captcha Library

### ISSUE THREAD

**Issue by sprankhub** [iMi-digital/magento2-friendly-captcha#9]
Currently, this module uses version 0.8.4:

https://github.com/iMi-digital/magento2-friendly-captcha/blob/fd34284fee7b842b961f340effa096ced841465b/view/frontend/web/js/vendor/friendly-challenge/package.json#L3

However, according to the [changelog](https://docs.friendlycaptcha.com/#/changelog), the latest version is 0.9.9. Especially this entry from version 0.9.0 sounds interesting:

> The widget now solves in multiple threads at the same time. This means that for the same computational cost a user has to wait shorter.

**Comment by amenk** [comment]
Yep, we should do this. Should be as easy as replacing the JS code :-) Feel free to make a PR.

**Comment by sprankhub** [comment]
See #12.

**Comment by amenk** [comment]
thanks again, will tag 1.2.0
